cql3/Cql.g

```diff
@@ -1012,14 +1012,8 @@ alterTableStatement returns [std::unique_ptr<alter_table_statement> expr]
     }
     ;
 
-cfisStatic returns [bool isStaticColumn]
-    @init{
-        bool isStatic = false;
-    }
-    : (K_STATIC { isStatic=true; })?
-    {
-        $isStaticColumn = isStatic;
-    }
+cfisStatic returns [bool isStaticColumn=false]
+    : (K_STATIC { $isStaticColumn=true; })?
```

Comment:
That's much better thanks. I didn't know that initialization syntax is supported

Reply:
+1